org.mixedrealitytoolkit.audio/AssemblyInfo.cs

```diff
@@ -9,4 +9,4 @@
 // The AssemblyVersion attribute is checked-in and is recommended not to be changed often.
 // https://docs.microsoft.com/troubleshoot/visualstudio/general/assembly-version-assembly-file-version
 // AssemblyFileVersion and AssemblyInformationalVersion are added by pack-upm.ps1 to match the current MRTK build version.
-[assembly: AssemblyVersion("3.0.0.0")]
+[assembly: AssemblyVersion("3.0.1.0")]
```

Comment:
> This is a manual fix for generating the docs.

Is this a temporary manual fix? Or something we're going to need to go going forward?

Reply:
> This is a temporary fix only needed for this release, so docs binaries have the correct versions.   We have fix this correct for main in this pr 
> 
> https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/638

Reply:
Excellent! Thanks for clarifying